src/extension.ts

```diff
@@ -31,7 +31,9 @@ import { TableAccountTreeItem } from './table/tree/TableAccountTreeItem';
 export function activate(context: vscode.ExtensionContext) {
 	context.subscriptions.push(new Reporter(context));
 
-	const tree: AzureTreeDataProvider = new AzureTreeDataProvider(new CosmosDBAccountProvider(), 'cosmosDB.loadMore', [new AttachedAccountsTreeItem(context.globalState)], reporter);
+	const ui: IAzureUserInput = new AzureUserInput(context.globalState);
+
+	const tree: AzureTreeDataProvider = new AzureTreeDataProvider(new CosmosDBAccountProvider(), 'cosmosDB.loadMore', ui, reporter, [new AttachedAccountsTreeItem(context.globalState)]);
```

Comment:
Looks like the order of parameters has changed, with `reporter` called before the array. Would this new signature break other calls across the extensions?

Reply:
Yes its a breaking change. That's why we bumped the minor version for the vscode-azureextensionui package from 8 to 9